strr-web/composables/useRegistrations.ts

```diff
@@ -36,9 +36,10 @@ export const useRegistrations = () => {
     axiosInstance.get(`${apiURL}/registrations/${id}`)
       .then(res => res.data)
 
-  const getDocumentsForRegistration = (id: string): Promise<DocumentI[]> =>
-    axiosInstance.get(`${apiURL}/registrations/${id}/documents`)
-      .then(res => res.data)
+  // TODO Remove this
```

Comment:
@dimak1 I did this to get the registration-details page working. I couldn't leave it uncommented as the url pattern for other endpoint is also the same. I believe you will be removing this as part of the UI changes.